src/services/public_http_server/handlers/relay_webhook/handlers/notify_subscribe.rs

```diff
@@ -129,6 +132,14 @@ pub async fn handle(msg: RelayIncomingMessage, state: &AppState) -> Result<(), R
             "request_auth.shared_claims.iss: {:?}",
             request_auth.shared_claims.iss
         );
+
+        request_auth
+            .validate()
+            .map_err(RelayMessageServerError::NotifyServer)?; // TODO change to client error?
```

Comment:
Can you elaborate on why it should be a client error?

Reply:
I have a bunch of these TODOs throughout the codebase, basically means "this is shitty error handling, you should re-evaluate if you can do it better".

In this case, it should be a client error because the request failed validation.